secilc/patches/disable-manpages.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/Makefile b/Makefile
```

Comment:
we can just `rm -rf` on the man folder rather than including a patch. or does it not work at all?

Reply:
Ah, we needed xml2man for that, so I decided to patch out building that altogether

Reply:
ah okay,  guess we have to live with this patch, wished this was configure based